samplePDF/samplePDFFDBase.cpp

```diff
@@ -763,15 +799,16 @@ void samplePDFFDBase::CalcXsecNormsBins(int iSample){
 		}
 		if (!ModeMatch) {continue;}
 
-		//ETA - this bit needs to change.....
 		//Now check whether the norm has kinematic bounds
 		//i.e. does it only apply to events in a particular kinematic region?
+		bool InBound = true;
 		if ((*it).hasKinBounds) {
 		  for (unsigned int iKinematicParameter = 0 ; iKinematicParameter < (*it).KinematicVarStr.size() ; ++iKinematicParameter ) {
-			if (ReturnKinematicParameter((*it).KinematicVarStr[iKinematicParameter], iSample, iEvent) < (*it).Selection[iKinematicParameter][0]) {continue;}
-			else if (ReturnKinematicParameter((*it).KinematicVarStr[iKinematicParameter], iSample, iEvent) > (*it).Selection[iKinematicParameter][1]) {continue;}
+			if (ReturnKinematicParameter((*it).KinematicVarStr[iKinematicParameter], iSample, iEvent) < (*it).Selection[iKinematicParameter][0]) {InBound=false;}
+			else if (ReturnKinematicParameter((*it).KinematicVarStr[iKinematicParameter], iSample, iEvent) > (*it).Selection[iKinematicParameter][1]) {InBound=false;}
 		  } 
 		}
+		if (!InBound) {continue;}
```

Comment:
was there any reason you've changed this? It feels like we now have an extra if for no reason.

Reply:
Originally the continue was breaking out of the kinematic loop but not the event loop, so kinematic cuts weren't being applied. Setting this InBound variable then breaks out of the event loop too